src/CoreBundle/private/library/classes/TCMSFields/TCMSField.class.php

```diff
@@ -745,10 +745,11 @@ public function _GetSQLDefinition(&$postData = null)
         $connection = $this->getDatabaseConnection();
         $inputFilterUtil = $this->getInputFilterUtil();
 
-        $lengthSet = $inputFilterUtil->getFilteredInput('length_set');
+        $lengthSet = '';
+        $cmsFieldTypeId = '';
+        $fieldDefaultValue = '';
 
         if (null !== $postData) {
```

Comment:
Rename this variable?

It is highly confusing with respect to the special case $inputFilterUtil...

Reply:
Done. It is now called `fieldDefinition`